web/modules/custom/private_message/joinup_private_message.info.yml

```diff
@@ -0,0 +1,9 @@
+type: module
+name: 'Joinup Private Messages'
+description: 'Basic private messaging infrastructure.'
+core: 8.x
+package: Joinup
+
+dependencies:
+  - message_private
+  - message_thread
```

Comment:
The message_thread module requires message_history and that is not included in composer.json so it is not installed in our builds :/

Reply:
I don't get it, is installing just fine for me, check `composer.lock:11402` with this branch. Even, you don't see a dependency  to `message_history` in the `composer.json` of `message_thread` module, the dependency is still added. This happens because we're using the d.o "packagist" and that is smart enough to parse the module's `*.info.yml` file for dependencies and will add them as Composer dependencies. This is the reason why `message_history` landed in `composer.lock` and is correctly installed. However, missing the dependency in the module Composer file is still a bug but is not affecting.

If your build lacks `message_history` is because of a potential wrong Git merge (?)